javaee/api/src/main/java/org/jboss/forge/addon/javaee/servlet/SecurityConstraintDescriptor.java

```diff
@@ -0,0 +1,177 @@
+package org.jboss.forge.addon.javaee.servlet;
+
+import org.jboss.forge.addon.javaee.security.TransportGuarantee;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Descriptor of the security constraint element part of the web.xml
+ *
+ * @author <a href="mailto:ivan.st.ivanov@gmail.com">Ivan St. Ivanov</a>
+ */
+public class SecurityConstraintDescriptor
```

Comment:
https://github.com/forge/javaee-descriptors<div>
</div><div>
</div><div>-------- Mensagem original --------</div><div>De: "Ivan St. Ivanov" <notifications@github.com> </div><div>Data: 20/07/2015  14:56  (GMT-03:00) </div><div>Para: forge/core <core@noreply.github.com> </div><div>Cc: George Gastaldi <ggastald@redhat.com> </div><div>Assunto: Re: [core] FORGEPLUGINS-152: Add some JavaEE security commands (#570) </div><div>
</div>> @@ -0,0 +1,177 @@
> +package org.jboss.forge.addon.javaee.servlet;
> +
> +import org.jboss.forge.addon.javaee.security.TransportGuarantee;
> +
> +import java.util.ArrayList;
> +import java.util.List;
> +
> +/**
> + * Descriptor of the security constraint element part of the web.xml
> + *
> + * @author <a href="mailto:ivan.st.ivanov@gmail.com">Ivan St. Ivanov</a>
> + */
> +public class SecurityConstraintDescriptor

What is javaee-descirptors? :)

---
Reply to this email directly or view it on GitHub:
https://github.com/forge/core/pull/570/files#r35023357